js/ui/autosuggest.js

```diff
@@ -3,133 +3,12 @@
  * parameter information and live documentation.
  */
 window.ScratchpadAutosuggest = {
-    /**
-     * Initializes the autosuggest functionality and adds/modifies the
-     * completers to be applicable to KA.
-     */
-    init: function(editor) {
```

Comment:
The init() and enableLiveCompletion() code all revolve around the live-completion functionality, and do not relate to the documentation pop-up.

(In the React PR, I turned the rest of this file into a React component)